angularjs/angular.d.ts

```diff
@@ -717,7 +716,6 @@ declare module angular {
     ///////////////////////////////////////////////////////////////////////////
     interface IBrowserService {
         defer: angular.ITimeoutService;
-        [key: string]: any;
```

Comment:
Same as with Window. I'd rather force the user extend IBrowserService with this rule and make him feel a bit dirty and hackish, than providing this by default.